src/transaction/log_page_buffer.c

```diff
@@ -6727,6 +6728,9 @@ logpb_checkpoint_topops (THREAD_ENTRY * thread_p, LOG_INFO_CHKPT_SYSOP * &chkpt_
 LOG_PAGEID
 logpb_checkpoint (THREAD_ENTRY * thread_p)
 {
+  if (get_server_type () == SERVER_TYPE_PAGE)
+    return NULL_LOG_PAGEID;
```

Comment:
- The preferred style is to make scopes obvious.
- Add a comment to explain this is temporary
```suggestion
  if (get_server_type () == SERVER_TYPE_PAGE)
    {
      // todo: reactivate once checkpoint is changed without log records
      return NULL_LOG_PAGEID;
    }
```